perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -47,5 +47,9 @@ sub users_can_hide { return 1; }
 
 sub default_show_name { 0 }
 
+sub reports_ordering {
+    return { -desc => 'created' };
```

Comment:
Could we use `confirmed` rather than `created`? Created is really an internal date field not exposed externally, confirmed is the date the thing appears on the site and the date given as the "creation" of this report publicly. The order will normally be the same, but it's possible someone takes a while to confirm their report and then going by created order would look a bit wrong.